src/app/parcels/ActionBar/Statuses.tsx

```diff
@@ -16,10 +16,12 @@ import { fetchParcelStatuses } from "@/app/parcels/fetchParcelTableData";
 export type StatusType = ParcelStatus[][number];
 
 const nonMenuStatuses: StatusType[] = [
-    "Shipping Labels Downloaded",
-    "Shopping List Downloaded",
+    "Driver Overview Downloaded",
+    "Map Generated",
     "Out for Delivery",
     "Request Deleted",
+    "Shipping Labels Downloaded",
+    "Shopping List Downloaded",
```

Comment:
Shall we add comments next to each item as to why they aren't displayed?

Reply:
The only one I'm not sure of is "Out for Delivery"

Reply:
sure - happy to leave it as is for now